padglobal/padglobal.py

```diff
@@ -858,7 +892,10 @@ def which_to_text(self):
     def _get_emojis(self):
         emojis = list()
         for server_id in self.settings.emojiServers():
-            emojis.extend(self.bot.get_guild(int(server_id)).emojis)
+            try:
+                emojis.extend(self.bot.get_guild(int(server_id)).emojis)
```

Comment:
The int here shouldn't be necessary, but why are you wrapping this in a try/catch block anyway? What error are you expecting?

Reply:
Deleted Emoji Server.  It just keeps happening throwing errors at me and it's annoying tbh.  it's not necessary per se

Reply:
Sorry if I'm being thick but I'm not sure I understand the problem. Isn't this something you can fix just by unsetting your emoji server list?

Reply:
yes, it is.  it's just a safeguard in case it gets unset in prod or smth.  better to have unrendered awoken skills than non-working commands imo